.github/workflows/build.yml

```
@@ -20,67 +27,49 @@ jobs:
+              if: runner.os == 'Linux'
               uses: awalsh128/cache-apt-pkgs-action@latest
               with:
                   packages: llvm-20 llvm-20-dev libpolly-20-dev clang-20
                   version: 1.0
+            - name: Install LLVM dependencies from Homebrew (macOS)
+              if: runner.os == 'macOS'
+              run: |
+                  brew install llvm@20
+                  echo "LLVM_SYS_201_PREFIX=$(brew --prefix llvm@20)" >> $GITHUB_ENV
             - name: Set up Rust
               uses: actions-rs/toolchain@v1
```

actionlint (1.7.9)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)